iotdb/src/main/java/org/apache/iotdb/db/conf/directories/strategy/SequenceStrategy.java

```diff
@@ -53,7 +49,7 @@ public int nextFolderIndex() throws DiskSpaceInsufficientException {
 
   private int tryGetNextIndex(int start) throws DiskSpaceInsufficientException {
     int index = (start + 1) % folders.size();
-    while (!hasSpace(folders.get(index))) {
+    while (!FileUtils.hasSpace(folders.get(index))) {
```

Comment:
suppose there is no space any more, then start == -1, and then,.... the process will never finish the while-loop...

Reply:
Ok... seems that you guarantee that there must be available space by super.init(). If so, please add comments in the init().

Reply:
added.